### ISSUE TITLE
G910 

### ISSUE THREAD

**Issue by xFooL** [SpoinkyNL/Artemis#381]
Hello!

Since I got a new PC the  Hollow Wave Key Press Effect won't work on my G910 Orion Spark anymore. I tried it again with my laptop where I used my G910 before and it worked perfectly there. The settings are exactly the same so I really have no clue what I did wrong or what's not working. :/ I hope somebody can help me. :)

**Comment by SpoinkyNL** [comment]
Hi there, if you've got any version of Artemis installed could you uninstall it and try it with the latest version?
You can find the download link [over here](https://github.com/SpoinkyNL/Artemis/releases/download/1.11.0.0/artemis-setup-1.11.0.0.msi). Please let me know how it goes, thanks!

(If you chose not to reply, I'll close this issue in 7 days to clean up a bit :) )

**Comment by xFooL** [comment]
Thanks for the reply! I actually got it to work again a few days ago and I totally forgot about this post... sorry! :( Well I re-installed Artemis after I cleaned my PC (and changed the USB Ports of some devices including my G910 aswell) and it worked again. I really don't know what's different this time (except the different USB Port, which I am certain I tested back then...) but I'm glad it works as intended! Thanks for the help!
